### DISCUSSION TITLE
Just finished the Blockchain basics section

### DISCUSSION THREAD

**Discussion by codeBigInt** [Cyfrin/foundry-full-course-cu#3094] [General]
Twas a really cool and informative session, although I found it hard to understand the Rollup section of the course.

I would really appreciate a link any material that explains this for very easy assimilation.

Much love for the course 😊😊🤓

**Comment by EngrPips** [comment]
Hey @codeBigInt, I think this Youtube video https://www.youtube.com/watch?v=BgCgauWVTs0 will help.

**Reply by codeBigInt** [reply]
Thanks man

**Reply by codeBigInt** [reply]
Thanks @EngrPips twas a really great explanatory video 😊

Are u taking the course as well?

**Reply by EngrPips** [reply]
I completed the course many months ago.
